src/content/blog/drupalcon-portland-wrapup.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Drupalcon Portland 2024 Wrapup"
+pubDate: 2024-05-14
+# modifiedDate: 2024-04-23
+summary: Drupalcon Portland 2024 Wrapup
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/05/platform.sh-booth-with-ddev-block.png
+  alt: Platform.sh Booth With "Proud Sponsor of DDEV" block
+categories:
+  - Community
+---
+
+We go to Drupalcons because they're a great way to stay in touch with one of our valued communities. DDEV has deep roots in the Drupal world, and it's always a delight to see our stakeholders at Drupalcon and hear their successes and frictions.
+
+**Drupal 11**: Drupal is about to release its newest version, Drupal 11. DDEV already has explicit support for Drupal 11, but there have been some late frictions with requirements like Sqlite 3.45 and Yarn v4. However, DDEV has them all supported.
+
+**Drupal Starshot**: The founder of Drupal, Dries Buytaert, did his regular "Driesnote" and launched a new initiative called "[Starshot](https://www.drupal.org/starshot)" to build a more usable out-of-the-box experience with Drupal. This initiative will proceed in parallel with normal Drupal core development and may have an impact on DDEV support for Drupal.
```

Comment:
what kind of impact? good, bad? more or less funding?

Reply:
Ah, I was saying here that it's likely to force us to jump around in our support of Drupal Starshot, not talking about funding in any way.

Reply:
```suggestion
**Drupal Starshot**: The founder of Drupal, Dries Buytaert, did his regular "Driesnote" and launched a new initiative called "[Starshot](https://www.drupal.org/starshot)" to build a more usable out-of-the-box experience with Drupal. This initiative will proceed in parallel with normal Drupal core development and may have an impact on DDEV's Drupal support, as Drupal 11 has also forced us to be ready for new requirements.
```